core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -251,6 +251,12 @@ public void setCivicUrl(String property) {
             }
         }
     }
+    
+    public static final String USE_CUSTOM_DRIVER_ANNOTATIONS = "oncoprint.custom_driver_annotation.default";
+    public static final String BINARY_CUSTOM_DRIVER_ANNOTATION_MENU_LABEL = "oncoprint.custom_driver_annotation.binary.menu_label";
+    public static final String CLASS_CUSTOM_DRIVER_ANNOTATION_MENU_LABEL = "oncoprint.custom_driver_annotation.class.menu_label";
+    public static final String SHOW_BINARY_CUSTOM_DRIVER_ANNOTATION = "oncoprint.custom_driver_annotation.binary.show";
```

Comment:
We discussed it again with @sandertan and @Sjoerd-van-Hagen, and we decided to define a default name for both columns. Therefore, we need the "SHOW" attributes.